doc/source/merging.rst

```diff
@@ -857,19 +857,18 @@ The left frame.
    X = X.astype(CategoricalDtype(categories=['foo', 'bar']))
 
    left = pd.DataFrame({'X': X,
-                        'Y': np.random.choice(['one', 'two', 'three'], size=(10,))})
+                        'Y': np.random.choice(['one', 'two', 'three'],
+                                              size=(10,))})
    left
    left.dtypes
 
 The right frame.
 
 .. ipython:: python
 
-   right = pd.DataFrame({
-        'X': pd.Series(['foo', 'bar'],
-                       dtype=CategoricalDtype(['foo', 'bar'])),
-        'Z': [1, 2]
-   })
+   right = pd.DataFrame({'X': pd.Series(['foo', 'bar'],
+                        dtype=CategoricalDtype(['foo', 'bar'])),
+                        'Z': [1, 2]})
```

Comment:
not sure if this is pep8 correct, but it's misleading. The `dtype` seems to belong to `DataFrame` and not `Series`.

Reply:
Oops...bumped ``dtype`` back in line with the ``Series`` open paren.